assessment_software/src/app/services/course.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CourseService {
+
+  constructor() { }
+
+  submitCourse(courseName: string, yearSemester: string) {
+    console.log(`courseName: ${courseName}, yearSemester: ${yearSemester}.`);
+  }
+}
```

Comment:
for app/services/ it is better to put files that concern the whole application. But for local services like course service, it Is better to keep it close to its component app/add-course/... 

Please move this course.service.ts file to its component.